src/snmalloc/backend_helpers/buddy.h

```diff
@@ -16,6 +16,7 @@ namespace snmalloc
   class Buddy
   {
     std::array<RBTree<Rep>, MAX_SIZE_BITS - MIN_SIZE_BITS> trees;
+    size_t empty_above = 0;
```

Comment:
A comment probably couldn't go amiss, especially noting that this is above in the sense of `>=`, not just `>`, yes?

Reply:
The invariant implies `>=`, I'll add a comment.